platform/osgi/platform-osgi-configadmin/src/main/java/org/codice/felix/cm/file/EncryptingPersistenceManager.java

```diff
@@ -53,25 +57,39 @@
 
   private static final Set<String> EXCLUDED_PROPERTIES = new HashSet<>();
 
-  // Not worth adding a dependency on felix's fileinstall module just for the constant
-  static final String FELIX_FILENAME = "felix.fileinstall.filename";
-
   static {
     EXCLUDED_PROPERTIES.add(SERVICE_PID);
     EXCLUDED_PROPERTIES.add(SERVICE_FACTORYPID);
     EXCLUDED_PROPERTIES.add(FELIX_FILENAME);
   }
 
+  private class InitOperation implements PrivilegedAction<EncryptionAgent> {
+    private final String passwordDirectory;
+
+    InitOperation(String passwordDirectory) {
+      this.passwordDirectory = passwordDirectory;
+    }
+
+    @Override
+    public EncryptionAgent run() {
+      return new EncryptionAgent(passwordDirectory);
+    }
+  }
```

Comment:
"Technically" we don't need this anymore because the security manager is no longer in the picture, but I highly recommend the working code remains.

Reply:
Follow-up reminder to update config installer's methods with privileged blocks.

Reply:
If we ever re-introduce the security manager, then there will be more comments for the installer which should extends its own privileges when accessing the files on disk via the AccessController.